_health-care/_js/components/form-elements/GrowableTable.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+
+class GrowableTable extends React.Component {
+  constructor(props) {
```

Comment:
Which begs the question of...what in the world has our code been doing with this line left out?